management/src/clusterm/manager/client_test.go

```diff
@@ -89,53 +100,59 @@ func (s *managerSuite) TestPostSuccess(c *C) {
 		url: baseURL,
 	}
 
+	var reqHostGroupBody bytes.Buffer
+	c.Assert(json.NewEncoder(&reqHostGroupBody).Encode(testReq1HostGroupBody), IsNil)
+
+	var testFlags ActionFlags
 	tests := map[string]struct {
 		expURLStr string
 		nodeName  string
-		extraVars string
+		flags     ActionFlags
 		exptdBody []byte
-		cb        func(name, extraVars string) error
+		cb        func(name string, flags ActionFlags) error
 	}{
 		"commission": {
 			expURLStr: fmt.Sprintf("http://%s/%s/%s", baseURL, PostNodeCommissionPrefix, testNodeName),
 			nodeName:  testNodeName,
-			extraVars: "",
-			exptdBody: []byte{},
+			flags:     ActionFlags{"", ansibleMasterGroupName},
+			exptdBody: reqHostGroupBody.Bytes(),
```

Comment:
@vvb , sorry I should have been more clearer.

I meant that we can add a separate case/entry like `commission-host-group` that just sets host-group but not extra-vars. The existing okReturner can still be used but just that we will be validating extra-vars and host-group separately. We can add `commission-host-group-extra-vars` to test a mix as well (which is what you have right now). But up to you, what you have works too.